BaseFilter/sections/specific.txt

```diff
@@ -566,6 +566,13 @@ $subdocument,third-party,denyallow=disqus.com,domain=welovemanga.one
 ! NOTE: Specific rules
 !
 !
+ww2-weapons.com###sgpb-popup-dialog-main-div-wrapper
```

Comment:
I think we can add to the general rules in the basic filter
ww2-weapons.com###sgpb-popup-dialog-main-div-wrapper
ww2-weapons.com##.sgpb-popup-overlay

Reply:
we can block the popup script. Only for ads used it is here.